.github/workflows/nextPush.yml

```diff
@@ -405,7 +407,7 @@ jobs:
 
       - name: Cypress - run "${{ matrix.cypress-folder }}" tests
         working-directory: next
-        run: yarn cypress run --spec "${{ matrix.cypress-folder }}"
+        run: yarn cypress run --browser chrome --spec "${{ matrix.cypress-folder }}"
```

Comment:
Currently, debugger networking feature is enabled only with chrome/chromium/canary browsers (https://docs.cypress.io/guides/guides/launching-browsers)